en/components/tree.md

```diff
@@ -0,0 +1,259 @@
+### None
+In the igx-tree by default node selection is disabled. So you can not select or deselect a row through interaction with the Tree UI, the only way to complete these actions is to use the provided API method.
+### Bi-State
+To enable bi-state node selection in the igx-tree just set the [selection]({environment:angularApiUrl}/classes/igxtreecomponent.html#selection) property to **BiState**. This will render a checkbox for every node.  Each node has two states - selected or not.  This mode supports multiple selection.
+```html
+<igx-tree [selection]="'BiState'">
+</igx-tree>
+```
+### Cascading
+To enable cascade node selection in the igx-tree just set the selection property to **Cascading**. This will render a checkbox for every node. 
+```html
+<igx-tree [selection]="'Cascading'">
+</igx-tree>
+```
+In this mode a parent's selection state entirely depends on the selection state of its children. When a parent has some selected and some deselected children, its checkbox is in an indeterminate state.
+
+## Keyboard Navigation
+Keyboard navigation in IgxTree provides a rich variety of keyboard interactions for the user. This functionality is enabled by default and allows to navigate through the nodes.
+
+The IgxTree navigation is compliant with W3C accesibility standards and convenient to use.
+
+**Key Combinations**
+
+ - <kbd>Arrow Down</kbd> - navigates to the next visible node. Marks the node as active. Does nothing if on the LAST node
+ - <kbd>Ctrl + Arrow Down</kbd> - navigates to the next visible node. Does nothing if on the LAST node
+ - <kbd>Arrow Up</kbd> - navigates to the previous visible node. Marks the node as active. Does nothing if on the FIRST node
+ - <kbd>Ctrl + Arrow Up</kbd> - navigates to the previous visible node. Does nothing if on the FIRST node
+ - <kbd>Arrow Left</kbd> - on an expanded parent node, collapses it. If on a child node, moves to its parent node.
+ - <kbd>Arrow Right</kbd> - on an expanded parent node, navigates to the first child of the node. If on a collapsed parent node, expands it.
+ - <kbd>Home</kbd> - navigates to the FIRST node
+ - <kbd>End</kbd> - navigates to the LAST visible node
+ - <kbd>Tab</kbd> - navigates to the next focusable element on the page, outside of the tree
+ - <kbd>Shift + Tab</kbd> - navigates to the previous focusable element on the page, outside of the tree
+ - <kbd>Space</kbd> - toggles selection of the current node. Marks the node as active.
+ - <kbd>Enter</kbd> - activate the focused node. If the node has link in it, open the link
+ - <kbd>*</kbd> - expand the node and all sibling nodes on the same level
+
+When selection is enabled, end-user selection of nodes is only allowed through the rendered checkbox. Since both selection types allow multiple selection, the following mouse + keyboard interaction is available
+
+ - <kbd>Click</kbd> - toggles selection of the node if selection is enabled. Otherwise, focuses the node
```

Comment:
This is a bit misleading, since `Click` on the node header **does not** select it.